### DISCUSSION TITLE
[分享] 使用 1Panel 一键部署 Memos

### DISCUSSION THREAD

**Discussion by wanghe-fit2cloud** [usememos/memos#2435] [Show and tell]
## [1Panel](https://github.com/1Panel-dev/1Panel) 应用商店支持一键安装Memos。

不仅使您能够轻松地部署和管理 Memos 项目，还支持后续的维护操作，如启动、停止、备份、升级等等。

无论您是新手还是经验丰富的开发者，[1Panel](https://github.com/1Panel-dev/1Panel) 应用商店将成为您的得力助手，为您提供无与伦比的便捷性和可靠性。不再需要复杂的配置和手动操作，Memos项目的管理将变得轻而易举。

> 加入我们，体验一键化的Memos管理，提高您的工作效率！

**Comment by ghost** [comment]
经过体验后，我发现确实非常方便。
同时，通过 1Panel 能够轻松管理服务器、容器等运维工作，真是一举两得。

**Comment by vychodlc** [comment]
为什么应用商店安装完成之后访问服务器ip+5230端口没东西呀？

**Reply by wanghe-fit2cloud** [reply]
安装应用时，勾选对外暴露端口。
